buildSrc/src/main/java/org/elasticsearch/gradle/precommit/ThirdPartyAuditTask.java

```diff
@@ -115,19 +124,36 @@ public File getJarExpandDir() {
         );
     }
 
-    public void setExcludes(String... classes) {
-        excludes.clear();
+    public void allowMissingClasses(String... classesOrPackages) {
+        for (String each : classesOrPackages) {
+            missingClassExcludes.add(each);
+        }
+    }
+
+    public void allowViolations(String... violatingClasses) {
```

Comment:
Instead of "allow", I would use the term "ignore". I think that better matches what we are doing.

Reply:
I agree, I changed it in all the methods.